dev/gulpfile.js

```diff
@@ -1,2 +1,50 @@
 /* eslint-disable import/no-extraneous-dependencies */
-require('require-dir')('./gulp-tasks');
+
+const path = require('path');
+
+const gulp = require('gulp');
+const sourcemaps = require('gulp-sourcemaps');
+
+const postcss = require('gulp-postcss');
+const autoprefixer = require('autoprefixer');
+const pxtorem = require('postcss-pxtorem');
+
+const STATIC = path.resolve(__dirname, '..', 'static');
+const CSS = path.resolve(STATIC, 'target', 'stylesheets');
+const JS = path.resolve(STATIC, 'src', 'javascripts-legacy');
+const TRANSPILED = path.resolve(STATIC, 'transpiled', 'javascripts');
```

Comment:
@sndrs is it possible to move the transpiled folder under `target` so that IntelliJ will ignore it?

This is driving me crazy:

![image](https://cloud.githubusercontent.com/assets/52038/21853541/6e7433a6-d80f-11e6-81e1-78a982d333a8.png)

...there seems to be a 50/50 chance the transpiled version comes first, and when I don't notice that, I end up editing the transpiled version, _not_ the one I want under source control. This gets frustrating :crying_cat_face: